traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -205,6 +206,13 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 
 		//System
 		{1.2, http.MethodGet, `system/info/?(\.json)?$`, systeminfo.Handler(d.DB), auth.PrivLevelReadOnly, Authenticated, nil},
+
+		//Tenants
+		{1.3, http.MethodGet, `tenants/?(\.json)?$`, api.ReadHandler(tenant.GetRefType(), d.DB), auth.PrivLevelReadOnly, Authenticated, nil},
```

Comment:
I think you should change these to 1.2 because you are simply rewriting the same perl endpoints in go